### PR TITLE
Update other-teammate

### PR DESCRIPTION
**Pull request by krichdev**

### CODE REVIEW

File: other-teammate [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Here is some code a different teammate has worked on. 
+Here is some code a different teammate has worked on. Adding awesome stuff.
 
 Manager knows Engineer wants to get better at XYZ because of botany.
```
Comment (botany-lead) [inline other-teammate:3]: @botany-demo here is a good case to look into at how to solve X

Comment (botany-lead) [reply other-teammate:3]: @krichdev testing

Comment (botany-demo) [reply other-teammate:3]: @botany-lead hello

Comment (botany-demo) [reply other-teammate:3]: @krichdev you too

Comment (botany-demo) [reply other-teammate:3]: @krichdev again

Comment (botany-demo) [reply other-teammate:3]: @krichdev 1 more time

Comment (botany-demo) [reply other-teammate:3]: @krichdev hello

**Review by botany-lead — APPROVED** [review]
good idea

**Review by botany-demo — APPROVED** [review]
I like this

**Pull request merged**